src/MainProcess.py

```diff
@@ -20,15 +19,15 @@ def __init__(self, base_url, llm_model_name, embedding_llm_model_name, api_key):
     def get_main_llm(self):
         if self.main_llm is None:
             self.main_llm = ChatNVIDIA(base_url=self.base_url, model=self.llm_model_name, api_key=self.api_key, temperature=0)
-            # main_llm = OpenAI(base_url=self.base_url, model=self.llm_model_name, api_key="dummy_key", temperature=0,
-            #                    top_p=0.01)
+            main_llm = OpenAI(base_url=self.base_url, model=self.llm_model_name, api_key="dummy_key", temperature=0,
+                               top_p=0.01)
         return self.main_llm
 
     def get_embedding_llm(self):
         if self.embedding_llm is None:
             self.embedding_llm = HuggingFaceEmbeddings(
-                model_name="/Users/jnirosha/Projects/morpheus/all-mpnet-base-v2",
-                model_kwargs={'device': 'mps'},
+                model_name="sentence-transformers/all-mpnet-base-v2",
+                model_kwargs={'device': 'cpu'},
```

Comment:
This is also depend on the system. In my laptop, I have to change this line to `mps` which is for Apple Mac M4 cores. So, try to decide it from `os` or `sys` modules which device value it should have. Because if the machine has GPU cards, we should use `gpu` instead of `cpu`. On macOS we have `mps`.

Reply:
Make sure to add a print statement relates to embedding model details include what device its using. This is important when we deploy on OCP so that we know what resources we are actually using.